chain/cosmos/cosmos_chain.go

```diff
@@ -315,8 +333,12 @@ func (c *CosmosChain) ExportState(ctx context.Context, height int64) (string, er
 }
 
 // Implements Chain interface
-func (c *CosmosChain) CreatePool(ctx context.Context, keyName string, contractAddress string, swapFee float64, exitFee float64, assets []ibc.WalletAmount) error {
-	return c.getFullNode().CreatePool(ctx, keyName, contractAddress, swapFee, exitFee, assets)
+func (c *CosmosChain) CreatePool(ctx context.Context, keyName string, params ibc.PoolParams) (string, error) {
```

Comment:
`CreatePool` is now part of the Chain interface? I think the doc comment is stale.

Reply:
yes removed it yesterday from the ChainInterface, will clean up